components-core/src/main/java/com/adyen/checkout/components/model/payments/request/EContextPaymentMethod.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 7/6/2022.
```

Comment:
Copyright needs updating, there's a bunch of other files like this as well

Reply:
Ah wait, did you actually create these files that long ago? 😅

Reply:
Yes I did 😄